platform/golden/debian-11.7.pkr.hcl

```diff
@@ -0,0 +1,55 @@
+# This file was autogenerated by the 'packer hcl2_upgrade' command. We
+# recommend double checking that everything is correct before going forward. We
+# also recommend treating this file as disposable. The HCL2 blocks in this
+# file can be moved to other files. For example, the variable blocks could be
+# moved to their own 'variables.pkr.hcl' file, etc. Those files need to be
+# suffixed with '.pkr.hcl' to be visible to Packer. To use multiple files at
+# once they also need to be in the same folder. 'packer inspect folder/'
+# will describe to you what is in that folder.
+
+# Avoid mixing go templating calls ( for example ```{{ upper(`string`) }}``` )
+# and HCL2 calls (for example '${ var.string_value_example }' ). They won't be
+# executed together and the outcome will be unknown.
+
+# source blocks are generated from your builders; a source can be referenced in
+# build blocks. A build block runs provisioner and post-processors on a
+# source. Read the documentation for source blocks here:
+# https://www.packer.io/docs/templates/hcl_templates/blocks/source
+source "virtualbox-iso" "autogenerated_1" {
+  boot_command         = ["<esc><wait>", 
+    "install", 
+    " initrd=/install/initrd.gz", 
+    " debian-installer=en_US auto locale=en_US kbd-chooser/method=us", 
+    " fb=false debconf/frontend=noninteractive", 
+    " keyboard-configuration/variant=USA console-setup/ask_detect=false ", 
+    " keyboard-configuration/modelcode=SKIP keyboard-configuration/layout=USA", 
+    " auto-install/enable=true", 
```

Comment:
this looks interesting.

Reply:
A "working" solution is in the bullseye.pkr.hcl file, so I'd ignore these